benchmark/wildtime_benchmarks/data_generation_arxiv.py

```diff
@@ -17,19 +15,25 @@ def main():
     ArXivDownloader(args.dir).store_data(args.all, args.dummyyear)
 
 
+corrupted_idx_dict = {
+    2007: [33213],
+    2008: [22489],
+    2009: [64621, 165454],
+    2015: [42007, 94935],
+    2016: [111398],
+    2019: [41309, 136814],
+    2020: [102074],
+    2021: [32013, 55660]
+}
```

Comment:
there are some lines in the file that are corrupted, i.e. the csv file wrapper cannot properly read the data. This is to remove those lines.

Reply:
Can we add a brief comment here in the file explaining this? And thanks for catching this! Not sure why this did not come up before.